yo/db.py

```diff
@@ -21,10 +21,10 @@
                 'mention', 'follow', 'vote', 'comment_reply', 'post_reply',
                 'account_update', 'message', 'receive')
 
-TRANSPORT_TYPES = ('email', 'sms', 'wwwpoll')
+TRANSPORT_TYPES = ('email', 'sms', 'wwwpoll', 'mock')
```

Comment:
mock/testing features shouldn't be in production code

Reply:
Please note that there's no actual implementation of the mock transport here, just a value in the list of valid transport types. Perhaps "mock" could be replaced with "unknown" or similar?

For now, i'll rewrite the test suite to patch the TRANSPORT_TYPES variable.